src/main/java/blackjack/domain/gamer/GameParticipants.java

```diff
@@ -0,0 +1,65 @@
+package blackjack.domain.gamer;
+
+import blackjack.domain.GameRule;
+import blackjack.domain.result.GameStatistics;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+public class GameParticipants {
+
+    private final Dealer dealer;
+    private final List<Player> players;
+
+    private GameParticipants(Dealer dealer, List<Player> players) {
+        this.dealer = dealer;
+        this.players = players;
+    }
+
+    public static GameParticipants of(List<Player> players, Dealer dealer) {
+        return new GameParticipants(dealer, players);
+    }
+
+    public void dealInitialCards() {
+        List<GameParticipant> gameParticipants = getGameParticipants();
+
+        for (int i = 0; i < GameRule.INITIAL_DEALING_CARD_COUNT.getValue(); i++) {
+            gameParticipants.forEach(dealer::dealCard);
+        }
+
+        dealer.hideCard();
+    }
+
+    public void executeHitPhase() {
+        players.forEach(participant -> {
+                    while (participant.shouldHit()) {
+                        dealer.dealCard(participant);
+                        participant.showHand();
+                    }
+                }
+        );
+
+        while (dealer.shouldHit()) {
+            dealer.dealCard(dealer);
+        }
+
+        dealer.openHiddenCard();
+    }
+
+    public GameStatistics calculateGameStatistics() {
+        return GameStatistics.from(this);
+    }
+
+    public List<GameParticipant> getGameParticipants() {
+        return Stream.concat(Stream.of(dealer), players.stream())
+                .toList();
+    }
+
+    public Dealer getDealer() {
+        return dealer;
+    }
+
+    public List<Player> getPlayers() {
+        return players;
```

Comment:
외부에서 변경되지 않도록 해볼까요? ㅎㅎ
```suggestion
        return Collections.unmodifiableList(players);
```